blurring_as_a_service/performace_evaluation_pipeline/source/evaluate_with_cvt_metrics.py

```diff
@@ -0,0 +1,30 @@
+from blurring_as_a_service.metrics.metrics_utils import (
+    generate_binary_mask,
+    parse_labels,
+)
+
+
+def process_image_labels(labels):
+
+    true_classes, true_bboxes = parse_labels(labels["true"])
```

Comment:
This entire function or a really similar version seems to be repeated in metric utils.
Can you check and refactor to avoid repetition if it logically makes sens?